docs/consume-packages/Package-Restore.md

```diff
@@ -1,242 +1,222 @@
+NuGet Package Restore restores all of a project's dependencies that are listed in either a project file or a *packages.config* file. You can restore packages at any time with `nuget restore`, `dotnet restore`, or through Visual Studio. The `dotnet build` and `dotnet run` commands automatically restore packages, and you can configure Visual Studio to restore packages automatically when it builds a project.
 
-Package Restore makes sure that all a project's dependencies are available, without having to store them in source control. To configure your source control repository to exclude the package binaries, see [Packages and source control](../consume-packages/packages-and-source-control.md). 
+To promote a cleaner development environment and to reduce repository size, Package Restore makes all of a project's dependencies available without having to store them in source control. To configure your source control repository to exclude package binaries, see [Packages and source control](../consume-packages/packages-and-source-control.md).
 
-## Package Restore overview
+## Package Restore behavior
 
-Package Restore first installs the direct dependencies of a project as needed, then installs any dependencies of those packages throughout the entire dependency graph.
-
-If a package isn't already installed, NuGet first attempts to retrieve it from the [cache](../consume-packages/managing-the-global-packages-and-cache-folders.md). If the package isn't in the cache, NuGet tries to download the package from all enabled sources in the list at **Tools** > **Options** > **NuGet Package Manager** > **Package Sources** in Visual Studio. During restore, NuGet ignores the order of package sources, and uses the package from whichever source is first to respond to requests. For more information about how NuGet behaves, see [Common NuGet configurations](Configuring-NuGet-Behavior.md). 
-
-> [!Note]
-> NuGet doesn't indicate a failure to restore a package until all the sources have been checked. At that time, NuGet reports a failure for only the last source in the list. The error implies that the package wasn't present on *any* of the other sources, even though errors aren't shown for each of those sources individually.
-
-## Restore packages
+Package Restore tries to install all package dependencies to the state that matches the `<PackageReference>`s in a project file, such as *.csproj*, or `<package>`s in a *packages.config* file. In Visual Studio **Solution Explorer**, you can see the package references under **Dependencies** > **Packages**.
```

Comment:
Only PackageReference projects have their packages listed in the SolutionExplorer. 
Should we tweak this to clarify it or at least to not mistakenly suggest that packages.config has the same capability.